ui/pages/onboarding-flow/recovery-phrase/confirm-srp-modal.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  AlignItems,
+  IconColor,
+  TextAlign,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import {
+  Box,
+  Modal,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+  ButtonSize,
+  Button,
+  ButtonVariant,
+  Icon,
+  IconSize,
+  IconName,
+} from '../../../components/component-library';
+
+type ConfirmSrpModalProps = {
+  onContinue: () => void;
+  onClose: () => void;
+  isError: boolean;
+};
+
+export default function ConfirmSrpModal({
+  onContinue,
+  onClose,
+  isError,
+}: ConfirmSrpModalProps) {
+  const t = useI18nContext();
+
+  const handleContinue = () => {
+    if (isError) {
+      onClose();
+    } else {
+      onContinue();
+    }
+  };
+
+  return (
+    <Modal
+      isOpen
+      onClose={() => null}
```

Comment:
We should clean this up, surely there could be a better option; could this just be null?  Will this show a "x" that won't work?